interfaces/vrep/DQ_SerialVrepRobot.m

```diff
@@ -0,0 +1,242 @@
+            % This method constructs an instance of a DQ_SerialVrepRobot.
+            % Usage:
+            %     DQ_SerialVrepRobot(base_robot_name, robot_dof, robot_name, vrep_interface);  
+            %          base_robot_name: The base name of the robot in the CoppeliaSim scene.
+            %          robot_dof: The number of DoF of the robot in the CoppeliaSim scene.
+            %          robot_name: The name of the robot in the CoppeliaSim scene.
+            %          vrep_interface: The DQ_VrepInterface object connected to the CoppeliaSim scene.
+            %
+            % Example: 
+            %     vi = DQ_VrepInterface();
+            %     vi.connect('127.0.0.1',19997);
+            %     vrep_robot = DQ_SerialVrepRobot("my_robot", 7, "my_robot#1", vi);
+            %     
+            %     Note that the name of the robot should be EXACTLY the same as in the CoppeliaSim
+            %     scene. For instance, if you drag-and-drop a second robot, its name will become 
+            %     "my_robot#0", a third robot, "my_robot#1", and so on.
+
+            obj.robot_name = robot_name;
+            obj.vrep_interface = vrep_interface;
+
+            %% The use of 'initialize_joint_names_from_vrep()', as is done in the C++ implementation, is not supported on a constructor in MATLAB
+            % From the second copy of the robot and onward, VREP appends a
+            % #number in the robot's name. We check here if the robot is
+            % called by the correct name and assign an index that will be
+            % used to correctly infer the robot's joint labels.
+            splited_name = strsplit(obj.robot_name,'#');
+            robot_label = splited_name{1};
+            if ~strcmp(robot_label, base_robot_name)
+                error('Expected %s', base_robot_name)
+            end
+            if length(splited_name) > 1
+                robot_index = splited_name{2};
+            else
+                robot_index = '';
+            end
+            
+            % Initialize joint names, link names, and base frame name
+            obj.joint_names = {};
+            for i=1:robot_dof
+                current_joint_name = {robot_label,'_joint',int2str(i),robot_index};
+                obj.joint_names{i} = strjoin(current_joint_name,'');
+            end
+            obj.base_frame_name = obj.joint_names{1};
+        end
+
+        function joint_names = get_joint_names(obj)
+            % This method gets the joint names of the robot in the CoppeliaSim scene.
+            % Usage:
+            %     get_joint_names;
+            %
+            % Example: 
+            %     vi = DQ_VrepInterface();
+            %     vi.connect('127.0.0.1',19997);
+            %     vrep_robot = DQ_SerialVrepRobot("my_robot", 7, "my_robot#1", vi);
+            %     joint_names = vrep_robot.get_joint_names;
+
+            joint_names = obj.joint_names;
+        end
+
+        function set_configuration_space_positions(obj, q)
```

Comment:
Please change those methods according to my comments above.

Reply:
Names changed in [8a60f03](https://github.com/dqrobotics/matlab/pull/108/commits/8a60f03ab59b96016931954e1f8b119ee9bf1843).